.github/workflows/ci.yaml

```diff
@@ -47,9 +47,18 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest]
         bazelversion: ${{ fromJSON(needs.matrix-prep-bazelversion.outputs.bazelversions) }}
+        bzlmodEnabled: [true, false]
         folder:
           - "."
-          - "e2e/workspace"
+          - "e2e/smoke"
+        exclude:
+          # Don't test bzlmod with Bazel 5 (not supported)
+          - bazelversion: 5.3.2
+            bzlmodEnabled: true
+          # Broken due to stardoc usage, see
+          # https://github.com/bazelbuild/stardoc/issues/117#issuecomment-1380744704
+          - folder: .
+            bzlmodEnabled: true
```

Comment:
An alternative could be to set [`--deleted_packages=docs`](https://bazel.build/reference/command-line-reference#flag--deleted_packages). That will exclude `//docs` but will still be able to build and test the rest on CI.

Reply:
That would be nice... but there's no way to make that flag conditional on bzlmod so we'd have to introduce a --config like https://github.com/aspect-build/rules_js/commit/48d0228c8b0ac0df4eddef0780ee44d88207aa44#diff-874728f39b9bba99bc8a2949132b868dd74225faaa504ae43810a45516eaa613R76 (which has that complexity for exactly the same reason)

I hear commits are landing in Bazel to fix stardoc so I'd rather keep this simple and wait for a proper fix, I think